### DISCUSSION TITLE
需求AI插件工具文心一言配置的详细步骤  总提示未授权

### DISCUSSION THREAD

**Discussion by htwins019** [halo-dev/halo#5527] [General]
### System information

Java
IBM Semeru Runtime Open Edition / 21.0.2+13-LTS
数据库
MySQL Community Server - GPL / 8.1.0
操作系统
Linux 3.10.0-1127.el7.x86_64 / amd64

### What is the project operation method?

Docker

### What happened?

文章中调用 ai 工具请求时提示未授权
希望能提供详细的配置步骤

**Comment by LIlGG** [comment]
之前 Halo 应用商店的说明文档没有更新，你可以看看最新的文档。

https://www.halo.run/store/apps/app-riNgb

AI 工具只需要你提供文心一言的 API Key 和 Secret Key 即可。因此大部分其他操作都是在文心一言。
